src/editor.vue

```diff
@@ -0,0 +1,213 @@
+<template>
+  <main>
+    <div id="app">
+      <oc-modal
+        :title="$gettext('Save As')"
+        :button-cancel-text="$gettext('Cancel')"
+        :button-confirm-text="$gettext('Save')"
+        :has-input="true"
+        :input-label="$gettext('New filename')"
+        :input-description="
+          $gettext('Please enter filename to which this document should be stored.')
+        "
+        :input-value="fileName"
+        class="oc-mb-l uk-position-absolute"
+        :hidden="hideSaveAs"
+        @cancel="cancelSaveAs"
+        @confirm="confirmSaveAs"
+      />
+      <div id="mainContainer" />
+    </div>
+  </main>
+</template>
+
+<script lang="ts">
+import axios from 'axios'
+import {
+  useAccessToken,
+  useAppDefaults,
+  useConfigurationManager,
+  useStore
+} from '@ownclouders/web-pkg'
+import { defineComponent, ref } from 'vue'
+
+export default defineComponent({
+  setup() {
+    const store = useStore()
```

Comment:
Results in error `TypeError: Cannot read properties of null (reading 'proxy')` because `getCurrentInstance()` (= used deep down inside the `useStore` composable) is null... any idea how to work around that? 🤔

Reply:
https://github.com/owncloud/richdocuments/pull/496/commits/35b1822351416529ad45eefdecd71b17ea4ee7d7 🙈

Reply:
Impressively awful 😅 thank you!